src/scripts.js

```diff
@@ -1,213 +1,172 @@
-// import './css/base.scss';
-// import './css/styles.scss';
-
-// import recipeData from './data/recipes';
-// import ingredientData from './data/ingredients';
-// import users from './data/users';
-
-// import Pantry from './pantry';
-// import Recipe from './recipe';
-// import User from './user';
-// import Cookbook from './cookbook';
-// import domUpdates from './domUpdates';
-
+import './css/base.scss';
+import './css/styles.scss';
+
+import recipeData from './data/recipes';
```

Comment:
We can get rid of these imports from the data files since we are using fetch (lines 4-6)